CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 
 ### Features
-- Add `renderComponent` function in the public API @mnajdova ([#503](https://github.com/stardust-ui/react/pull/503))
+- Add `createComponent` function in the public API @mnajdova ([#503](https://github.com/stardust-ui/react/pull/503))
```

Comment:
@mnajdova fyi

Reply:
Thanks, I made this change already in my PR, but let's merge it soon after whichever of these two PRs gets in first.